plutus-example/plutus-example.cabal

```diff
@@ -0,0 +1,155 @@
+  default-language:     Haskell2010
+
+  default-extensions:   NoImplicitPrelude
+                        OverloadedStrings
+                        ImportQualifiedPost
+
+  ghc-options:          -Wall
+                        -Wcompat
+                        -Wincomplete-record-updates
+                        -Wincomplete-uni-patterns
+                        -Wpartial-fields
+                        -Wredundant-constraints
+                        -Wunused-packages
+                        -fobject-code
+                        -fno-ignore-interface-pragmas
+                        -fno-omit-interface-pragmas
+
+  -- See Plutus Tx readme
+  if flag(defer-plugin-errors)
+      ghc-options: -fplugin-opt PlutusTx.Plugin:defer-errors
+
+common maybe-Win32
+  if os(windows)
+     build-depends:     Win32
+
+library
+  import:               base, project-config
+                      , maybe-Win32
+
+  if flag(unexpected_thunks)
+    cpp-options: -DUNEXPECTED_THUNKS
+
+  hs-source-dirs:       src
+
+  exposed-modules:      PlutusExample.AlwaysFails
+                        PlutusExample.AlwaysSucceeds
+                        PlutusExample.CustomDatumRedeemerGuess
+                        PlutusExample.DatumRedeemerGuess
+                        PlutusExample.MintingScript
+                        PlutusExample.ScriptContextChecker
+                        PlutusExample.Sum
+                        PlutusExample.Loop
+
+  build-depends:        aeson
+                      , bytestring
+                      , cardano-api
+                      , cardano-cli
+                      , cardano-ledger-alonzo
+                      , cardano-ledger-core
+                      , cardano-slotting
+                      , containers
+                      , ouroboros-consensus
+                      , ouroboros-network
+                      , plutus-ledger
+                      , plutus-ledger-api
+                      , plutus-tx
+                      , plutus-tx-plugin
+                      , serialise
+                      , strict-containers
+                      , transformers
+                      , transformers-except
+
+
+executable plutus-example
+  import:               base, project-config
+  hs-source-dirs:       app
+  main-is:              plutus-example.hs
+  ghc-options:          -threaded -rtsopts "-with-rtsopts=-T"
+
+  build-depends:        cardano-api
+                      , directory
+                      , filepath
+                      , plutus-example
+
+
+executable create-script-context
+  import:               base, project-config
+  hs-source-dirs:       app
+  main-is:              create-script-context.hs
+  ghc-options:          -threaded -rtsopts "-with-rtsopts=-T"
+
+  build-depends:        bytestring
+                      , cardano-api
+                      , optparse-applicative
+                      , plutus-example
+                      , transformers
+
+test-suite plutus-example-test
```

Comment:
Is this currently being run in CI?

Reply:
@asutherlandus  How do I add this to run in CI?

Reply:
I did run in CI: https://hydra.iohk.io/build/12955862